.github/scripts/lib/gate-reconciliation-core.ps1

```
@@ -0,0 +1,250 @@
+if ($IssueNumber -and $IssueNumber -gt 0) {
+    $tokens = $tokens | Where-Object { $_.issue_number -eq $IssueNumber -or -not $_.issue_number }
+}
+
+# 2. Read recorded engagement-record decisions (decision_id coverage)
+$erArgs = @{ IssueNumber = $IssueNumber; GhCliPath = $GhCliPath }
+if ($Phase)           { $erArgs.Phase = $Phase }
+if ($Repo)            { $erArgs.Repo = $Repo }
+if ($InMemoryMarkers) { $erArgs.InMemoryMarkers = $InMemoryMarkers }
+
+$engagementRecords = @()
+try { $engagementRecords = Read-EngagementRecords @erArgs } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)